clipping_manager/migrations/0014_auto_20210726_0917.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2021-07-26 09:17
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
The two migrations could be squashed, right? (you can do it by eg. rolling back, delete the files, run makemigrations again)